app/views/publishers/home.html.slim

```diff
@@ -180,8 +180,8 @@ script id="choose-channel-type" type="text/html"
         - if channel.verified?
           - if current_publisher.promo_status(promo_running?) == :active && channel.promo_enabled?
             .channel--promo-info-container
-              = link_to("", tweet_url(channel.promo_registration.referral_code), target: :_blank, class: "promo-share-button promo-share-button-twitter")
-              = link_to("", facebook_url(channel.promo_registration.referral_code), target: :_blank, class: "promo-share-button promo-share-button-facebook")
+              = link_to("", tweet_url(channel.promo_registration.referral_code), target: :_blank, class: "promo-share-button promo-share-button-twitter", :"data-piwik-action" => "ShareTwitterClicked", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "Dashboard")
+              = link_to("", facebook_url(channel.promo_registration.referral_code), target: :_blank, class: "promo-share-button promo-share-button-facebook", :"data-piwik-action" => "ShareFacebookClicked", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "Dashboard")
```

Comment:
@alexwykoff heads up, there are a number of places where display twitter and share link buttons (dashboard, promo page upon activation, and in the promo emails).  so this will only represent a subset of share buttons clicked.